packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -318,14 +316,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         if (!this.attributesBlobHandles.has(blobId)) {
```

Comment:
You could have removed this bloc as well (as it's noop), and likely - tracking of attributesBlobHandles, as it's no longer needed.

Reply:
oh i see!